src/posh-winfetch.ps1

```diff
@@ -430,4 +444,4 @@ write-output ''
 # | __/ _ \| __|
 # | _| (_) | _|
 # |___\___/|_|
-#
+#
```

Comment:
It probably doesn't matter, but it's always best to have a newline at the end of a file ;)

Reply:
That's a really good approach, :)